README.md

```diff
@@ -10,3 +10,10 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## My Response
+Two graphs that meet the requirements do not have to be fully connected.
+Let $A$ and $B$ both be graphs, and let them each contain only one node.  This means that they have no edges, and are 
+completely unconnected.
+then by the definition they are one to one and onto, which satisfies the bijection requirement.
```

Comment:
What is one to one and unto? A graph (or set of graphs) can't be.

Reply:
my apologies, I thought I answered this already before class on Tuesday, a graph can't be one to one or onto, what can, and as to be for our definition, is the mapping or the function, and all it's saying is that there is a graph A that maps onto B, and for each node in A there is a corresponding node in B, so by having two graphs who are both only one node, it the function map can be both one to one, onto, and both graphs can be completely disconnected.

Reply:
That's all great, but you need to say that in your proof. At the moment it says "they are one to one and onto", where "they" refers to the graphs. You need to explicitly talk about the mapping.